backend/actions/transaction_list.js

```diff
@@ -12,90 +12,44 @@ app.use((req, res, next) => {
   next();
 });
 
-async function getMemTx() {
-  const txids = await tapyrusd.getRawMempool();
-  const txs = await Promise.all(
-    txids.map(txid => electrs.blockchain.transaction.get(txid, true))
-  );
-
-  const txsWithTime = [];
-  for (let tx of txs) {
-    const txDetail = await tapyrusd.command([
-      {
-        method: 'getmempoolentry',
-        parameters: {
-          txid: tx.txid
-        }
-      }
-    ]);
-    tx.time = txDetail[0].time;
-
-    txsWithTime.push(tx);
-  }
-
-  return txsWithTime.sort((a, b) => b.time - a.time);
-}
-
 //Return a List of transactions
 app.get('/transactions', async (req, res) => {
```

Comment:
Should here be changed to `/txs` ?

Reply:
I feel like txs is an abbreviation of thanks. If I were to change it, it would be txns, but for clarity, I think it should be left as transactions.